js/src/ui/Page/page.css

```diff
@@ -18,7 +18,7 @@
 .layout {
   padding: 0.25em 0.25em 1em 0.25em;
 
-  > * {
-    margin-bottom: 0.75em;
+  &>div {
```

Comment:
So this is supposed to be a property of the `.page` class and not its descendants. That's why there is `padding: 0.25em 0.25em 1em 0.25em` for the `Page` Component. Anyway, looks good now :)